zerver/lib/actions.py

```diff
@@ -773,10 +773,13 @@ def do_send_messages(messages_maybe_none):
         Message.objects.bulk_create([message['message'] for message in messages])
         ums = []  # type: List[UserMessage]
         for message in messages:
-            # Outgoing webhook bots don't store UserMessage rows; they will be processed later.
```

Comment:
Would anyone happen to know why we don't need to store a `UserMessage` row here?